### DISCUSSION TITLE
Add the pre-built wheesl to pypi

### DISCUSSION THREAD

**Discussion by wqh17101** [open-mmlab/mmcv#2430] [General]
**Describe the feature**
Add the pre-built wheel to pypi
**Motivation**
It is better for you to add the pre-built wheesl to pypi that will make users convenient to download and install

**Related resources**
It seems that your packages have the same version for different cuda or cpu.

https://download.pytorch.org/whl/torch/
![image](https://user-images.githubusercontent.com/26429138/182999824-04950f75-9115-41e0-9b85-32e7584eee57.png)

Like torch, you can add + to version like mmcv-1.6.1+cpu / mmcv-1.6.1+cu102. So that users can choose the whl easily

**Additional context**
If you would like to implement the feature and create a PR, please leave a comment here and that would be much appreciated.

**Comment by zhouzaida** [comment]
Hi @wqh17101 , thanks for your suggestions. In fact, you can install mmcv-full with `openmim` which will be very convenient.
```bash
pip install -U openmim
mim install mmcv-full
```

**Comment by wqh17101** [comment]
@zhouzaida Yeah, that is  a good tool for now.
But as a system,there are many python packages, nobody wants to install a installer for every packages. All we need is a common one ——pip

**Comment by wqh17101** [comment]
@zhouzaida Any plan to do this enhancement

**Comment by zhouzaida** [comment]
Hi, like PyTorch, we also put pre-compiled packages on our host's service, e.g. https://download.openmmlab.com/mmcv/dist/cu102/torch1.8.0/index.html. We don't put all packages on PyPI because the MMCV packages have different combinations of torch and CUDA versions. Putting them on PyPI would require a new package for each torch and CUDA combination.

**Comment by wqh17101** [comment]
@zhouzaida Right.However there is an enhancement for you.
You can see that pytorch adds + to version etc:torch-1.8.0+cu110 to make it easy for recognizing where the artifact can be used.
For you , you can see that you only use the url to distinguish them.
ETC:
I download https://download.openmmlab.com/mmcv/dist/cu102/torch1.10.0/mmcv_full-1.6.1-cp38-cp38-manylinux1_x86_64.whl  and https://download.openmmlab.com/mmcv/dist/cu102/torch1.11.0/mmcv_full-1.6.1-cp38-cp38-manylinux1_x86_64.whl . These are two different packages with the same package name.
So,it is hard for users to distinguish them after downloading.

There is an advice for you.
Use different contents after + to distinguish them.
like this:
mmcv_full-1.6.1+cu102.torch1.11.0-cp38-cp38-manylinux1_x86_64.whl for version 1.6.1+cu102.torch1.11.0
mmcv_full-1.6.1+cu102.torch1.10.0-cp38-cp38-manylinux1_x86_64.wh for version 1.6.1+cu102.torch1.10.0
